src/types/hull-trait-name.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+
+/**
+ * Trait names are strings
+ */
+export type HullTraitNameType = string;
```

Comment:
Thinking forward, we wanted to be consistent and name all traits as attributes, so I would suggest to name it `HullAttributeNameType`.